public/client.js

```
@@ -0,0 +1,534 @@
+        exportableSpectrumData = filteredData;
+        
+        // Make export button visible
+        document.getElementById('exportSpectrumBtn').style.display = 'inline-block';
+    }
+    
+    // Function to render integrated spectrum chart
+    function renderIntegratedSpectrumChart(frequencyBins, magnitudeValues, quantity) {
+        const ctx = document.getElementById('integratedSpectrumChart').getContext('2d');
+        
+        // Filter out frequencies above the max frequency
+        const maxFreq = parseFloat(document.getElementById('maxFrequency').value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)